nautobot/extras/tests/test_webhooks.py

```diff
@@ -101,9 +101,7 @@ class FakeResponse:
         with patch.object(Session, "send", mock_send):
             self.client.force_login(self.user)
 
-            change_context = ORMChangeContext(self.user, id=request_id)
-
-            with change_logging(change_context):
+            with web_request_context(self.user, change_id=request_id):
```

Comment:
Why change this back (and the similar ones below?)

Reply:
Just saving a few lines of code. They're functionally identical